app/Console/Commands/ImportSignupsCommand.php

```diff
@@ -14,7 +14,7 @@ class ImportSignupsCommand extends Command
      *
      * @var string
      */
-    protected $signature = 'rogue:signupimport {path}';
+    protected $signature = 'rogue:signupimport {path} {logfreq=1000}';
```

Comment:
This is so nitpicky, but I think this feels like it might make more sense as an [option](https://laravel.com/docs/5.5/artisan#options):

```sh
php artisan rogue:signupimport example.csv --logfreq=1000
# versus:
php artisan rogue:signupimport example.csv 1000
```

Reply:
@DFurnes I like that better too!